jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -175,10 +175,12 @@
    */
   private Tracer tracer;
 
+  @Deprecated
```

Comment:
does this still allow populating Configuration object from a config file?

Reply:
Ignore this, I want to say that only `Tracer.Builder builder = Configuration.fomEnv()` should be used. See PR description